extensions/objects/templates/FilepathEditor.js

```diff
@@ -581,6 +581,7 @@ oppia.directive('filepathEditor', [
         };
 
         $scope.setSavedImageFilename = function(filename, updateParent) {
+          var dimensions = $scope.calculateTargetImageDimensions();
```

Comment:
Could this be moved just before line 594? Because we seem to need the dimensions only on that line.

Reply:
Done!

Reply:
No we can't actually. Because the calculateTargetImageDimension uses the $scope.data in MODE_UPLOADED to get the dimensions. If we place it before line 594, them it can't actually get the dimensions of the image, since the $scope.data gets into MODE_SAVED and doesn't have the dimensions values

Reply:
Ok